Directory.Build.props

```diff
@@ -3,31 +3,11 @@
 <Project>
 
   <PropertyGroup>
-    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
-
-  <Import Project="build\NuGet.props"/>
-  <Import Project="build\Versions.props"/>
-
-  <PropertyGroup>
-    <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
-    <RepoToolsetDir>$(NuGetPackageRoot)roslyntools.repotoolset\$(RoslynToolsRepoToolsetVersion)\tools\</RepoToolsetDir>
-    <RestoreSources>
-      https://dotnetfeed.blob.core.windows.net/dotnet-core/packages/index.json;
-      https://dotnet.myget.org/F/nuget-build/api/v3/index.json
-    </RestoreSources>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <ArtifactsDir>$(DOTNET_SDK_ARTIFACTS_DIR)</ArtifactsDir>
-    <ArtifactsDir Condition="'$(ArtifactsDir)' == ''">$(RepoRoot)artifacts\</ArtifactsDir>
```

Comment:
Is this set by RepoToolset now?

Reply:
DOTNET_SDK_ARTIFACTS_DIR is no longer needed.

---
In reply to: [184185770](https://github.com/dotnet/sdk/pull/2178#discussion_r184185770) [](ancestors = 184185770)

Reply:
What do you mean, it is still used by our tests for dogfooding.

Reply:
That still needs to be cleaned up (I'll follow up on that later). It's used as an optional override of the artifacts directory path here:

[https://github.com/dotnet/sdk/blob/9cf7788e45687df15dae183dc9277ac690528062/src/Tests/Microsoft.NET.TestFramework/TestContext.cs#L100-L104](https://github.com/dotnet/sdk/blob/9cf7788e45687df15dae183dc9277ac690528062/src/Tests/Microsoft.NET.TestFramework/TestContext.cs#L100-L104)

Based on discussion with @dsplaisted the scenario this was needed for was to switch to a different version of dotnet SDK. This can be done by `DOTNET_INSTALL_DIR` instead. So `DOTNET_SDK_ARTIFACTS_DIR` is not needed anymore.

---
In reply to: [184195832](https://github.com/dotnet/sdk/pull/2178#discussion_r184195832) [](ancestors = 184195832)